packages/template-retail-react-app/app/commerce-api/auth.js

```diff
@@ -478,12 +483,16 @@ class Auth {
         }
         const response = await this._api.shopperLogin.getAccessToken(options)
         // Check for error response before handling the token
+        // Note: SLAS getAccessToken will only return a status code on error
+        // The status code will be a string with a format like `400 BAD_REQUEST`
         if (response.status_code) {
-            // if the refresh has failed, clear auth so the user is
+            // if the refresh login has failed, clear auth so the user is
             // not stuck with an invalid refresh token
             this._clearAuth()
-            console.log(`Auth Error: HTTP ${response.status_code} - ${response.message}`)
-            throw new HTTPError(response.status_code, response.message)
+            console.error(`Auth Error: HTTP ${response.status_code} - ${response.message}`)
+
+            // fall back to new guest login
+            return this._loginAsGuest()
```

Comment:
Minor: how about us `console.log`ing here to tell others that we've handled the error by doing a guest login? I think it'd be useful for them to see that in the log too... not just the errors.